src/zenml/cli/stack.py

```diff
@@ -681,3 +688,125 @@ def down_stack(force: bool = False) -> None:
             f"Suspending resources for active stack '{stack_.name}'."
         )
         stack_.suspend()
+
+
+@stack.command("export")
+@click.argument("filename", type=str, required=False)
+@click.argument("stack_name", type=str, required=False)
+def export_stack(filename: Optional[str], stack_name: Optional[str]) -> None:
+    """Export a stack to YAML."""
+
+    # Get configuration of given stack
+    # TODO: code duplicate with describe_stack()
+    repo = Repository()
+
+    active_stack_name = repo.active_stack_name
+    stack_name = stack_name or active_stack_name
+
+    if not stack_name:
+        cli_utils.warning("No stack is set as active!")
+        return
+
+    stack_configurations = repo.stack_configurations
+    if len(stack_configurations) == 0:
+        cli_utils.warning("No stacks registered!")
+        return
+
+    try:
+        stack_configuration = stack_configurations[stack_name]
+    except KeyError:
+        cli_utils.error(f"Stack '{stack_name}' does not exist.")
+        return
```

Comment:
Is this necessary? The `cli_utils.error` raises an exception.

Reply:
True, removed the redundant return, thanks!